src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,426 @@
+= Vert.x for Groovy
+
+If you are using Maven or Gradle, add the following dependency to the _dependencies_ section of your
+project descriptor to access the Vert.x Core API and enable the Groovy support:
+
+* Maven (in your `pom.xml`):
+
+[source,xml,subs="+attributes"]
+----
+<dependency>
+ <groupId>io.vertx</groupId>
+ <artifactId>vertx-core</artifactId>
+ <version>${maven.version}</version>
+</dependency>
+<dependency>
+  <groupId>io.vertx</groupId>
+  <artifactId>vertx-lang-groovy</artifactId>
+  <version>${maven.version}</version>
+</dependency>
+----
+
+* Gradle (in your `build.gradle` file):
+
+[source,groovy,subs="+attributes"]
+----
+compile "io.vertx:vertx-core:${maven.version}"
```

Comment:
Is it really ${maven.version} here?

Reply:
yes, this is replaced by the actual version by the doc tool